src/App.jsx

```diff
@@ -1,13 +1,156 @@
+        });
+    };
+    return (
+        <div className="fruit-container">
+            <form onSubmit={handleSubmit} className="order-form">
+                <div className="section fruit-section">
+                    <h1>Fruitmand bezorgservice</h1>
+                    <Counter
+                        label="🍌 Bananen"
+                        value={bananas}
+                        onIncrease={() => setBananas(bananas + 1)}
+                        onDecrease={() => setBananas(Math.max(0, bananas - 1))}
+                    />
+                    <Counter
+                        label="🍓 Aardbeien"
+                        value={strawberries}
+                        onIncrease={() => setStrawberries(strawberries + 1)}
+                        onDecrease={() => setStrawberries(Math.max(0, strawberries - 1))}
+                    />
+                    <Counter
+                        label="🍏 Appels"
+                        value={apples}
+                        onIncrease={() => setApples(apples + 1)}
+                        onDecrease={() => setApples(Math.max(0, apples - 1))}
+                    />
+                    <Counter
+                        label="🥝 Kiwi&apos;s"
+                        value={kiwis}
+                        onIncrease={() => setKiwis(kiwis + 1)}
+                        onDecrease={() => setKiwis(Math.max(0, kiwis + 1))}
+                    />
+                    <Button text="Reset alles" onClick={reset}>Reset alles</Button>
+                </div>
+
+
+                <div className="section order-form-section">
+                    <TextInput
+                        label="Voornaam:"
+                        value={firstName}
+                        onChange={setFirstName}
+                    />
+                    <TextInput
+                        label="Achternaam:"
+                        value={lastName}
+                        onChange={setLastName}
+                    />
+                    <TextInput
+                        label="Leeftijd:"
+                        value={age}
+                        onChange={setAge}
+                        type="number"
+                    />
+                    <TextInput
+                        label="Postcode:"
+                        value={postcode}
+                        onChange={setPostcode}
+                    />
+
+                    <label>
+                        Bezorgfrequentie:
+                        <select
+                            value={deliveryFrequency}
+                            onChange={(e) => setDeliveryFrequency(e.target.value)}
+                        >
+                            <option value="iedere week">Iedere week</option>
+                            <option value="om de week">Om de week</option>
+                            <option value="iedere maand">Iedere maand</option>
+                        </select>
+                    </label>
+
+                    <RadioGroup
```

Comment:
radio is ook een type van het input element. Je zou dus een global input component kunnen maken die dusdanig aanpasbaar is a.d.h.v. properties dat je er zowel een text als een radio input van kunt maken.